src/SFVBolivia/Helpers/Bill.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SFVBolivia.Helpers
+{
+    public class Bill
+    {
+        private int billNumber;
+        public int BillNumber { get; set; }
+
+        private long authorization;
+        public long Authorization { get; set; }
+
+        private DateTime date;
+        public DateTime Date { get; set; }
+
+        private double amount;
+        public double Amount { get; set; }
+
+        private double amountFiscalCredit;
+        public double AmountFiscalCredit { get; set; }
+
+        private string controlCode;
+        public string ControlCode { get; set; }
+
+        private long nITRecep;
+        public long NITRecep { get; set; }
+
+        private UserIssuer userIssuer;
+        public UserIssuer UserIssuer { get; set; }
+
+
+        public Bill()
```

Comment:
Empty constructor is not necessary because of the next constructor, remove it

Reply:
done.